packages/cli/src/util/deploy/parse-target.ts

```diff
@@ -37,5 +37,12 @@ export default function parseTarget(
     return 'production';
   }
 
+  if (process.env.VERCEL_ENV) {
+    output.debug(
+      `Setting target to ${process.env.VERCEL_ENV} using VERCEL_ENV environment variable.`
+    );
+    return process.env.VERCEL_ENV;
```

Comment:
Should this env var be validated first?

Reply:
Yep, thanks. It's in a later commit 👇🏾 😄